projects/website/generate-news.js

```diff
@@ -105,11 +105,7 @@ function generateRoutes() {
     }
   });
   var final_routes = ROUTES_TEMPLATE.replace('${routes}', routes);
-  fs.writeFileSync(
-    FINAL_ROUTES_PATH,
-    `// tslint:disable
-  ${final_routes}`
-  );
+  fs.writeFileSync(FINAL_ROUTES_PATH, `// tslint:disable\n${final_routes}`.trim());
```

Comment:
I don't think you need `// tslint:disable` now as we don't use it any longer. We should resort to eslint-disable if necessary.